### ISSUE TITLE
How do I continue where I left off?

### ISSUE THREAD

**Issue by jaredmorgs** [mc-t1/mct1#33]
If I'm really enjoying the MC-T1 experience and I leave the hospital, then my journey is interrupted if I'm playing on borrowed/hospital laptops.

Is there a way that I could save my progress like a save file up to the Magikcraft server and then install and pick it up when I got home?

**Comment by jwulf** [comment]
We have `magik.durablePlayerMap` and now database-backing.

**Comment by jaredmorgs** [comment]
Sounds like this issue can be closed, providing we cover how to save in the User Docs or the UI.

**Comment by jwulf** [comment]
No, it needs to be kept open. I just mean to say: "We have the technology".

We still need to implement this feature. 

Obviously, it requires the hospital or borrowed laptop to have an internet connection, and for you to be playing with your own Minecraft account. 

Otherwise, we could do an export via web-browser to a file that you put on a USB and then upload via web-browser at home. But that's got to be 2.0, and only if we need it.

In 1.0 we'll just do persistent state attached to your minecraft identity.
